### PR TITLE
Add HHBtag_v2 model

### PR DESCRIPTION
**Pull request by elviramartinv**
This Pull Request is made to check the new HHBtag v2 model and setup  while continuing in parallel with stability studies

### CODE REVIEW

**Review by kandrosov — CHANGES_REQUESTED** [review]
There should be no changes to v1

**Comment by elviramartinv** [comment]
Now there are no changes to v1

**Comment by kandrosov** [comment]
thanks! Please also:
- add tests in `bin/hhBtagTest.cc` for v2 (keeping v1 as well)
- update documentation in README.md

**Comment by elviramartinv** [comment]
README and bin/hhBtagTest.cc are already updated, please let me know if anything else is missing

File: README.md [hunk 1]
```diff
@@ -4,8 +4,8 @@ Interface providing feature computation for CMS RunII HH-btag NN to the identifi
 # Installation
 
 ```sh
-cmsrel CMSSW_11_1_0_pre6 # or above
-cd CMSSW_11_1_0_pre6/src
+cmsrel CMSSW_13_0_0_pre1 # or above
+cd CMSSW_13_0_0_pre1/src
```
Comment (kandrosov) [inline README.md:8]:
```suggestion
cmsrel CMSSW_12_4_10 # or above
cd CMSSW_12_4_10/src
```

File: bin/hhBtagTest.cc [hunk 2]
```diff
@@ -1,12 +1,13 @@
 #include <map>
 #include "../interface/HH_BTag.h"
 
-int main()
+int main(int argc, char *argv[])
 {
+    std:string version = "v1" // model version "v1" or "v2"
```
Comment (kandrosov) [inline bin/hhBtagTest.cc:6]: - take the model version from the command line so that one can test both
- as it is now, the code doesn't compile

**Review by kandrosov — CHANGES_REQUESTED** [review]

**Review by kandrosov — APPROVED** [review]

**Comment by kandrosov** [comment]
Thank you @elviramartinv. All looks good. However, I will wait before merging: if training for the whole Run 2 arrives before the new skim iteration, I would prefer to merge it directly. Otherwise, I'll merge this version as v2 and then merge the full Run 2 training as v3.

**Pull request merged**